portfolio/src/main/webapp/index.html

```diff
@@ -7,8 +7,16 @@
     <script src="script.js"></script>
   </head>
   <body bgcolor = "#66819c">
+    <div id="navbar">
```

Comment:
FYI: There is a <nav> element specifically meant for such cases.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav